### PR TITLE
Feature ldap config

### PR DESCRIPTION
**Pull request by francois-dorin**
## Problem

LDAP authentication not configured by default

## Solution

Execute an SQL script at installation to create LDAP source.

## PR Status

- [X] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

File: conf/db-ldap.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+INSERT INTO  public.auth_sources(type, name, host, port, base_dn, attr_login, attr_firstname, attr_lastname, attr_mail, onthefly_register, filter) VALUES (
+	'AuthSourceLdap',
+        'yunohost',
+        'localhost',
+        389,
+        'ou=users,dc=yunohost,dc=org',
+        'uid',
+        'sn',
+        'givenName',
```
Comment (tevino) [inline conf/db-ldap.sql:9]: The order of `sn`(surname) and `givenName` needs a swap, they are currently mapped to `attr_firstname` and `attr_lastname`.

**Review by tevino — CHANGES_REQUESTED** [review]

File: conf/db-ldap.sql (hunk 1 above)
Comment (francois-dorin) [reply conf/db-ldap.sql:9]: You're right. I will fix this soon and update this PR.

Thank you for your feedback :)

Comment (tevino) [reply conf/db-ldap.sql:9]: I haven't tried the script in the PR but a manual configuration according to the sql works.

And thank you @francois-dorin for your work :)

Comment (francois-dorin) [reply conf/db-ldap.sql:8]: I fix the inversion and updated the PR

@tevino : thank you :)

**Comment by ericgaspar** [comment]
!testme

**Comment by OniriCorpe** [comment]
!testme

**Comment by OniriCorpe** [comment]
ah, i didn't see that it was already adde to the package by https://github.com/YunoHost-Apps/redmine_ynh/pull/23

so closing the PR

thank you, anyway ^w^

**Comment by francois-dorin** [comment]
Well, in fact my contribution seems to have been "secretly" pickup into PR #23 in a commit with message "cleaning" (cf. https://github.com/YunoHost-Apps/redmine_ynh/pull/23/commits/1fcae428eaeef8ab3ff52f18b33f4789d14f061d)

Is it a normal behavior in contribution managment ?

**Pull request closed without merging**